testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -6231,6 +6231,28 @@ void assertSubFunReturns(boolean binary, String s, int start,
     f0.forEachLibrary(list(SqlLibrary.BIG_QUERY, SqlLibrary.ORACLE), consumer);
   }
 
+  @Test void testLengthFunc() {
```

Comment:
I'd move this test next to `testCharLengthFunc` because it is so similar.

Use the `f0.forEachLibrary(list(SqlLibrary.BIG_QUERY), consumer);` pattern so that the test can be extended to other libraries in future.